docs/example_scripts/test_kg.py

```diff
@@ -31,7 +31,7 @@
 
 
 # get overall results of different metrics
-for name, metric_info in sys_info.results.overall[0].items():
+for name, metric_info in sys_info.results.overall["example"].items():
```

Comment:
We can consider put the global-level config variable `example` [here](https://github.com/neulab/ExplainaBoard/blob/main/explainaboard/config.py) in future PR.